pkg/lint/runner.go

```diff
@@ -111,10 +111,29 @@ func (r *Runner) runLinterSafe(ctx context.Context, lintCtx *linter.Context,
 		}
 	}()
 
+	// pkgs will get dirty while analyzing, which affects to other linters' result.
+	// To avoid this issue, we clone the loaded packages rather than directly using them.
+	oldPackages := lintCtx.Packages
+	oldOriginalPackages := lintCtx.OriginalPackages
+	clone := func(pkgs []*gopackages.Package) []*gopackages.Package {
```

Comment:
Also not 100% sure but it may be better to use `reflect.Copy`. See my example here... https://goplay.space/#WJWMX29euwU

Reply:
Thank you, but as I commented above, I will delete this change later 😭